From a8571ec98fba1f1ca0fe49326907960650754785 Mon Sep 17 00:00:00 2001
From: Robert Lee <robert.lee@freescale.com>
Date: Fri, 16 Oct 2009 09:41:55 +8000
Subject: [PATCH] ENGR00117297 cpufreq clock/voltage table needs to be updated

Previous values being used were for STMP378x which has lower temperature
requirements.  The updated values work with 105C temperatures but require
higher digital core voltages and lower maximum EMI clock speeds.:$lower
maximum EMI clock speeds.:

Signed-off-by: Robert Lee <robert.lee@freescale.com>
---
 arch/arm/plat-stmp3xxx/cpufreq.c |   20 ++++++++++----------
 1 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/arch/arm/plat-stmp3xxx/cpufreq.c b/arch/arm/plat-stmp3xxx/cpufreq.c
index aa83d33..4c7486b 100644
--- a/arch/arm/plat-stmp3xxx/cpufreq.c
+++ b/arch/arm/plat-stmp3xxx/cpufreq.c
@@ -56,17 +56,17 @@ static struct profile {
 	int vdda;
 	int pll_off;
 } profiles[] = {
-	{  24000,  24000,  24000, 3, 1000000,
-	925000, 150000, 3075000, 1725000, 1 },
-	{  64000,  64000,  48000, 3, 1000000,
+	{  24000,  24000,  24000, 3, 1050000,
+	975000, 150000, 3075000, 1725000, 1 },
+	{  64000,  64000,  48000, 3, 1050000,
 	925000, 150000, 3300000, 1750000, 0 },
-	{ 261820, 130910, 130910, 0, 1225000,
-	1125000, 173000, 3300000, 1750000, 0 },
-	{ 360000, 120000, 120000, 0, 1350000,
-	1250000, 200000, 3300000, 1750000, 0 },
-	{ 392730, 130910, 130910, 0, 1400000,
-	1300000, 225000, 3300000, 1750000, 0 },
-	{ 454740, 151580, 151580, 0, 1550000,
+	{ 261820, 130910, 130910, 0, 1275000,
+	1175000, 173000, 3300000, 1750000, 0 },
+	{ 360000, 120000, 120000, 0, 13750000,
+	1275000, 200000, 3300000, 1750000, 0 },
+	{ 392730, 130910, 130910, 0, 1475000,
+	1375000, 225000, 3300000, 1750000, 0 },
+	{ 454740, 151580, 130910, 0, 1550000,
 	1450000, 355000, 3300000, 1750000, 0 },
 };
 
-- 
1.5.4.4

